wcomponents-theme/src/test/spec/wc.ui.printButton.test.mjs

```diff
@@ -0,0 +1,28 @@
+import "wc/ui/printButton.mjs";
+import domTesting from "@testing-library/dom";
+
+describe("wc/ui/printButton", () => {
+	const linkId = "ui-printbutton-1",
+		testContent = `<button class="wc-printbutton" data-testid="${linkId}" type="button"  >Click me<button>`;
+	let testHolder;
+
+	beforeAll(() => {
+		testHolder = document.body.appendChild(document.createElement("div"));
+		testHolder.innerHTML = testContent;
+	});
+
+	it("test", function() {
```

Comment:
Ok fair enough there is only one test but still... Maybe a slightly better name?
What is this test doing, perhaps something like:

it("sets preventDefault on the print button click event",